charts/timed/templates/configmap-backend.yaml

```diff
@@ -16,7 +16,7 @@ data:
   DJANGO_DATABASE_USER: {{ .Values.postgresql.postgresqlUsername | quote }}
   DJANGO_DATABASE_PORT: {{ .Values.postgresql.service.port | quote }}
   DJANGO_WORK_REPORT_PATH: {{ .Values.backend.settings.workReportPath | quote }}
-  DJANGO_OIDC_USERINFO_ENDPOINT: {{ .Values.auth.oidc.userinfoEndpoint | quote }}
+  DJANGO_OIDC_USERINFO_ENDPOINT: {{ printf "%s/userinfo" .Values.auth.oidc.url | quote }}
```

Comment:
This is specific to our using it with keycloak and we'll need to change it if we end up using a different IdP.